cmd/clusterctl/log/log.go

```diff
@@ -23,6 +23,7 @@ import (
 
 // SetLogger sets a concrete logging implementation for all deferred Loggers.
 func SetLogger(l logr.Logger) {
+	Log = log.NewDelegatingLogger(log.NullLogger{})
```

Comment:
This doesn't feel quite right, should this be 
```suggestion
	Log = log.NewDelegatingLogger(l)
```

But it seems like Log.Fulfill(l) alone should replace the null logger that was initialized below for `Log`

Reply:
This was my original assumption, but it is not working and apparently it is allowed to replace the default logger with a custom logger, but not to replace a custom logger with another logger...

> // Fulfill switches the logger over to use the actual logger
> // provided, instead of the temporary initial one, if this method
> // has not been previously called.

Reply:
What's the underlying cause? Are we calling SetLogger multiple times?